test/parallel/test-net-connect-local-error.js

```diff
@@ -3,25 +3,46 @@ const common = require('../common');
 const assert = require('assert');
 const net = require('net');
 
+const fixedPort = 8080;
```

Comment:
@Trott I thought to use it because of your comment on this: https://github.com/nodejs/node/pull/12639

Also as I understood from the `common.PORT` in [parallel tests issue](https://github.com/nodejs/node/issues/12376), it was planned to use a linting rule against using `common.PORT` in parallel tests.